charts/partners/fortanix/fortanix-secrets-injector/1.3/report.yaml

```diff
@@ -0,0 +1,89 @@
+apiversion: v1
+kind: verify-report
+metadata:
+    tool:
+        verifier-version: 1.1.0

```

Comment:
Can you run the latest verifier?: https://quay.io/repository/redhat-certification/chart-verifier?tab=tags
Version 1.1.0 is affected ubi8 CVE that was fixed already. Version 1.2.0 and latest contains the ubi8 layers with the fix.

Reply:
Version 1.2.1 I should have said

Reply:
The report is now regenerated with latest verifier